src/runtime/native/runtime.ml

```diff
@@ -31,6 +48,12 @@ let full_path fname =
 
 let env_to_fname env default = try Sys.getenv env with Not_found -> !default
 
+let env_to_boolean env default =
```

Comment:
```suggestion
let env_to_boolean env ?(default = false) =
```

Reply:
`Warning 16: this optional argument cannot be erased.`

Reply:
How's that possible? You have `env` as the last argument? (Before editing I did put `default` as the last argument).

Reply:
I messed up. Here's the fix: a7db78e